src/viewer.js

```diff
@@ -2845,72 +2811,72 @@ function onCanvasKeyPress( event ) {
                 break;
             case 119://w
             case 87://W
-                if (!canvasKeyPressEventArgs.preventVerticalPan) {
+                if (!canvasKeyDownEventArgs.preventVerticalPan) {
                     if ( event.shift ) {
                         this.viewport.zoomBy(1.1);
                     } else {
                         this.viewport.panBy(this.viewport.deltaPointsFromPixels(new $.Point(0, -40)));
                     }
-                    this.viewport.applyConstraints();
-                  }
-                  event.preventDefault = true;
-                  break;
+                        this.viewport.applyConstraints();
```

Comment:
This shouldn't be tabbed out, for instance.

Reply:
can you explain more what you mean by

Reply:
just a formatting issue, lines 2820-2083 should be moved 4 spaces to the left